x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/index.tsx

```diff
@@ -445,9 +445,12 @@ export const CreatePackagePolicyPage: React.FunctionComponent = () => {
           )}
 
           {/* If an Agent Policy and a package has been selected, then show UI extension (if any) */}
-          {ExtensionView && packagePolicy.policy_id && packagePolicy.package?.name && (
+          {extensionView && packagePolicy.policy_id && packagePolicy.package?.name && (
             <ExtensionWrapper>
-              <ExtensionView newPolicy={packagePolicy} onChange={handleExtensionViewOnChange} />
+              <extensionView.Component
```

Comment:
We had to make a similar change to this everywhere we use `useUIExtension` to accommodate the new signature wherein we return the entire `extension` object and no longer just the `component`. This also needs to be capitalized as enforced by the JSX spec.